### DISCUSSION TITLE
How to change the color of the block blame.

### DISCUSSION THREAD

**Discussion by mjaysona** [gitkraken/vscode-gitlens#2201] [Q&A] [answered]
How do I change the color of the label '**You, 2 minutes ago | 1 autho (You)**'. I've tried all the color customization stated in the docs but none worked.

![image](https://user-images.githubusercontent.com/14266979/188852432-c030456e-a898-4e38-bf46-a0009b4466ad.png)

**Comment by mjaysona** [comment]
Up

**Comment by eamodio** [comment, marked as answer]
Thse are called CodeLens, and you can change the color by overriding the theme color via the `workbench.colorCustomizations`.

See below to set the CodeLens to pink

```json
  "workbench.colorCustomizations": {
    "editorCodeLens.foreground": "#ff00ff"
  },
```

Hope that helps!

**Reply by mjaysona** [reply]
This worked. Thanks!
